.github/CODEOWNERS

```
@@ -0,0 +1,8 @@
+# Github code owners
+# Siehe auch https://help.github.com/articles/about-codeowners/
+#
+# Bitte die Sortierung dieser Datei beibehalten. Die zuletzt zutreffende Regel
+# hat Vorrang.
+
+# global codeowner
+* @andrecrema @tschreibereuropace 
```

```suggestion
* @andrecrema @tschreibereuropace @AlexanderKru 
```

Wenn niemand einen Einwand hat würde ich mich noch mit rein nehmen und sagen wir haben genug CodeOwner

---

ich trag dich mal noch mit ein